### PR TITLE
Connecting workout back front

### PR DESCRIPTION
**Pull request by Parshana007**

### CODE REVIEW

**Comment by haysho2260** [comment]
- are the binary files necessary? if not could you make sure to not push them to git. We also need to keep in mind if they're not necessary to not merge all with main
- code looks good though
- helpful error messages
- i think we should be consistent with out naming conventions because we mix up camel case and snake case

File: backend/src/api/workouts.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        final_array.append(
+            {
+                "date": exercise.date,
+                "lift_name": exercise.lift_name, 
+                "weight": exercise.weight,
+                "sets": exercise.sets,
+                "reps": exercise.reps
+            }
+        )
+    print(final_array)
```
Comment (joanna-chou) [inline backend/src/api/workouts.py:41]: Testing print statement

File: backend/src/api/workouts.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    if result is None:
+        # Raise an HTTPException with a 404 status code and an error message
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail="User not found")
+    
+    final_array = []
+    for exercise in result:
+        final_array.append(
+            {
+                "date": exercise.date,
+                "lift_name": exercise.lift_name,
```
Comment (joanna-chou) [inline backend/src/api/workouts.py:35]: Frontend "workoutName" but I like liftName more probably

File: react-frontend/src/components/Home.js [hunk 3]
```diff
@@ -1,61 +1,87 @@
...
-     const workoutCardList = result.map((card) => {
-          return (
-               <WorkoutCard
-                    key={card[0]}
-                    date={card[0]}
-                    item={card[1]}
-               ></WorkoutCard>
-          );
-     });
+     const workoutCardList = workoutData.map((card) => (
```
Comment (joanna-chou) [inline react-frontend/src/components/Home.js:73]: (For me to implement after merge) Sort exercises by date before here

File: react-frontend/src/components/WorkoutCard.js [hunk 4]
```diff
@@ -34,35 +19,32 @@ function WorkoutCard(props) {
...
-                                             <th style={{ width: "10%" }}>
-                                                  Weight
-                                             </th>
-                                             <th style={{ width: "10%" }}>
-                                                  Sets
-                                             </th>
-                                             <th style={{ width: "10%" }}>
-                                                  Reps
-                                             </th>
+                                             <td>{props.item.lift_name}</td>
```
Comment (joanna-chou) [inline react-frontend/src/components/WorkoutCard.js:36]: Won't display multiple weightlifts for a single day. May have to implementing sorting by dates and mapping (workoutData in Home.js )

**Comment by Parshana007** [comment]
- fixed camel cases to snake cases 
- I believe the binary files are necessary so keeping them for backend

**Comment by Parshana007** [comment]
- Amazing cool remembered to not commit the binary files and realized that they were unnecessary to commit as well as made sure to git ignore the env file for security reasons
- Thank you for the meaningful comments!

**Pull request merged**